cli/parse.go

```diff
@@ -79,6 +78,11 @@ func isRecursive(req *cmds.Request) bool {
 	return rec && ok
 }
 
+func resolveCommandLine(req *cmds.Request) bool {
+	linkOpt, ok := req.Options[cmds.DerefLong].(bool)
+	return linkOpt && ok
+}
```

Comment:
`linkOpt` will automatically be `false` if `ok` is `false`. So you could just 
```go
linkOpt, _ := ...
return linkOpt
```

Reply:
Shortened that and [another section](https://github.com/ipfs/go-ipfs-cmds/compare/75a3c41ad8827a64cba3bfb4aac7e142d002afcd..69aacbe1b3ebb44cd2b82e3dcba3e7c8fda3b622).

Reply:
@Stebalien 
Sorry about that, the semantic difference between casting the return from the map `if req.Options[cmds.DerefLong].(bool)`, and casting the return of a type assertion
`safeVar, _ := req.Options[cmds.DerefLong].(bool)` tripped me up.